src/arguments/CoreEmoji.ts

```diff
@@ -7,13 +7,14 @@ export class CoreArgument extends Argument<EmojiObject> {
 		super(context, { name: 'emoji' });
 	}
 
-	public run(parameter: string): ArgumentResult<EmojiObject> {
+	public run(parameter: string, context: Argument.Context): ArgumentResult<EmojiObject> {
 		const resolved = resolveEmoji(parameter);
 		return resolved.mapErrInto((identifier) =>
 			this.error({
 				parameter,
 				identifier,
-				message: 'The argument did not resolve to an emoji.'
+				message: 'The argument did not resolve to an emoji.',
+				context
```

Comment:
move this above message :)

Reply:
but that's not how [it's done](https://github.com/sapphiredev/framework/blob/800730381041bf9dad9ea389bff2de3dad791b78/src/arguments/CoreBoolean.ts#L16-L17)

Reply:
My lord